app/controllers/api/base_controller/manager.rb

```diff
@@ -2,8 +2,10 @@ module Api
   class BaseController
     module Manager
       def update_collection(type, id)
-        if @req.method == :put || @req.method == :patch
-          return send("#{@req.method}_resource", type, id)
+        if @req.method == :put
+          return edit_resource(type, id, @req.json_body)
```

Comment:
Should this be `put_resource`?

```suggestion
          return put_resource(type, id, @req.json_body)
```

Reply:
Oh this is the collapsing of the redirection you mentioned - didn't realize this is what you meant.